purescript.cabal

```diff
@@ -962,7 +964,7 @@ executable purs
 
 test-suite tests
     build-depends:
-          aeson >=1.0 && <1.2
+          aeson >=1.0 && <1.1
```

Comment:
Yes, this has happened too many times now; I think we do need to either gitignore the cabal file or stop using hpack. I'm not particularly attached to one option over the other but if it was solely up to me I'd probably go for stopping using hpack.